docs/README.md

```diff
@@ -2,70 +2,95 @@
 sidebar_position: 1
 ---
 
-# Welcome to Ignite documentation!
+# Ignite - the battle-tested React Native boilerplate
+
+With over seven years of active development, Ignite is the most popular React Native app starter boilerplate for both Expo and bare React Native.
+
+This is the React Native stack that the [Infinite Red](https://infinite.red) team uses on a day-to-day basis to build client apps. Developers who use Ignite report that it saves them two to four weeks of time on average off the beginning nd improves their confidence over the lifetime of the entire project.
 
 ## Getting Started
 
-### Quick Start
+If you just want to kick off a new project, you can do that with the Ignite CLI:
 
 ```terminal
-npx ignite-cli@latest new pizza-app --yes
+npx ignite-cli@latest new pizza-app
 ```
 
-### Full Guide
+Prerequisites:
+
+- Node.js LTS release or greater
+- Recommended: recent versions of Xcode and Android Studio
+- Recommended: Yarn or Bun package manager
+- Recommended: macOS (for iOS development)
 
-If you're new to Ignite, start by reading our [Getting Started Guide](./Guide.md)!
+### Intro to Ignite
 
-### Intro Video
+If you're new to Ignite, check out these resources:
 
 - Check out the [Getting Started with Ignite](https://www.youtube.com/watch?v=KOSvDlFyg20) video for a 13 minute overview!
-- Jamon's Code Quest on [Ignite 9](https://www.youtube.com/watch?v=QmkMsUYrTlk)
+- Jamon's Code Quest on [What's new in Ignite 9](https://www.youtube.com/watch?v=QmkMsUYrTlk)
+- Check out our [Guide](./Guide.md) for a walkthrough of the boilerplate project and how to use it
+
+<table>
+<tr>
+<td><img alt="ignite-01-sign-in" src="https://github.com/infinitered/ir-docs/assets/1479215/d84d5efa-363d-44bc-a7b3-6ec70d878ae3" /></td>
```

Comment:
where are these images? Im curious - I don't see them in either ir-docs or ignite repo, so I'm curious how they end up hosted here!